suse_migration_services/fstab.py

```diff
@@ -113,3 +116,27 @@ def export(self, filename):
 
     def get_devices(self):
         return self.fstab
+
+    def get_root_disk(self):
+        """Get the disk name for / partition."""
+        lsbkl_mountpoint_call = Command.run(
+            ['lsblk', '-r', '-n', '-o', 'NAME,MOUNTPOINT']
+        )
+
+        for entry in lsbkl_mountpoint_call.output.split(os.linesep):
+            block_record = entry.split()
+            if len(block_record) >= 2 and block_record[1] == '/':
+                lsbkl_disk_call = Command.run(
+                    [
+                        'lsbkl', '-n', '-o', 'pkname',
+                        '/dev/{}'.format(block_record[0])
+                    ]
+                )
+                return lsbkl_disk_call.output
+
+    def is_on_root(self, device_path):
+        """Check that the entry belongs to the root disk."""
+        lsblk_call = Command.run(
+            ['lsblk', '-n', '-o', 'pkname', device_path]
+        )
+        return lsblk_call.output == self.root_disk
```

Comment:
Looks good :+1:

Reply:
But would make the method also private: _is_on_root_disk(...)

Reply:
With the changes above the two methods export() and get_devices() needs to be adapted to write or return only devices that are _is_on_root_disk()